src/Api/Api.php

```diff
@@ -7,24 +9,27 @@
 abstract class Api
 {
     /**
-     * @var string HubSpot API key
+     * Hubspot API key
+     * @var string
      */
     protected $apiKey;
 
     /**
+     * HttpClient implementation
      * @var HttpClient
      */
     protected $client;
 
     /**
+     * Use OAuth?
      * @var bool
      */
     private $oauth;
 
     /**
-     * @var string Base url
+     * Base url
      */
-    protected $baseUrl = "https://api.hubapi.com";
+    const BASE_URL = "https://api.hubapi.com";
```

Comment:
I think it should stay as an instance variable, it is overridden in the [Events API](https://github.com/ryanwinchester/hubspot-php/blob/master/src/Api/Events.php#L7)